_data/terms.yml

```diff
@@ -18,14 +22,16 @@
  "Hydraulic fracturing": "A well development process that involves injecting water under high pressure into a bedrock formation through the well, to increase the size and extent of existing bedrock fractures."
  "Independent Administrator (IA)": "The EITI International Board requires participating countries to appoint an Independent Administrator to help apply the international standards. The USEITI Independent Administrator is Deloitte & Touche LLP."
  "Indian lands": "Lands owned by Native Americans, including tribal lands held in trust by the federal government for a tribe’s use, Indian allotments held in trust by the federal government for individual use, and lands held by Alaska Native corporations."
+ "Kilowatt hour (kWh)": "A measure of electrical energy equivalent to a power consumption of 1,000 watts for 1 hour; abbreviated as kWh."
+ "kWh": "Abbreviation for “kilowatt hour,” a measure of electrical energy equivalent to a power consumption of 1,000 watts for 1 hour."
  "Lease": "A contract that allows a company to be the exclusive entity that can apply to explore for and extract natural resources within a specific tract of federal lands or waters."
  "Lease condensate": "Light liquid hydrocarbons recovered from oil and natural gas wells during production."
  "Locatable minerals": "Locatable minerals are minerals that may be “located” and obtained by filing a mining claim. Locatable minerals include gold, silver, copper, lead, and many other metallic and nonmetallic minerals."
  "Margin of variance": "The percentage difference that the USEITI Multi-Stakeholder Group defined as significant for each revenue type as part of the reconciliation process."
  "Material variance": "A discrepancy between government-reported and company-reported revenue payments that is considered significant by the Independent Administrator. Margins of variance vary by revenue type, and were approved by the Multi-Stakeholder Group as part of the USEITI process."
  "Mcf": "1000 cubic feet, a unit of measure for natural gas."
  "Megawatt Capacity (MC) fee": "A revenue payment for the calculated value of electricity generated on federal lands."
- "Megawatt hours (Mwh)": "One megawatt is equivalent to one million watts. One megawatt hour is equivalent to 1,000 Kilowatt hours (Kwh)."
+ "Megawatt hours": "One megawatt is equivalent to one million watts. One megawatt hour (abbreviated as Mwh) is equivalent to 1,000 Kilowatt hours."
```

Comment:
Yeah — that's one option I thought about for #1988. Clearly I started going down that path here, but it makes me nervous because it gets much harder to maintain if any definitions change. If we implement per-unit glossary terms, I can come back and remove the duplicates I just introduced.